content/blog/top-5-things-for-azure-devs-devops/index.md

```diff
@@ -16,7 +18,14 @@ What comes to mind when you hear DevOps? Frequently, DevOps is described as a cu
 
 DevOps processes rely on tooling to automate delivering and maintaining applications. This article demonstrates how to implement Azure Pipelines to build and deploy applications.
 
-<!--more-->
+In this series guide, we've covered:
+* [Top Azure Services Every Developer Should Know: Overview](/blog/top-5-things-for-azure-devs-intro/)
```

Comment:
this seems super repetitive does it? "Top Azure Services Every Developer Should Know:" 8 times?

Can't we just link to the tag page and put the tag in order like this:

https://www.pulumi.com/blog/tag/platform-engineering-pillars/